### DISCUSSION TITLE
New postcss config format confusion

### DISCUSSION THREAD

**Discussion by Christophvh** [tailwindlabs/tailwindcss#2712] [Help] [answered]
After i did following command: `npx tailwindcss init -p`

I get this postcss.config.js file:
```
module.exports = {
    plugins: {
        tailwindcss: {},
        autoprefixer: {},
    },
}
```

But I'm used to adding those 2 plugins like this:
```
module.exports = {
  plugins: [
    // ...
    require('tailwindcss'),
    require('autoprefixer'),
    // ...
  ]
}

```
Is the above the same or am I missing something?

**Comment by simonswiss** [comment, marked as answer]
Hi!

They are both essentially the same. As far as I know, the Object syntax has better compatibility with external tooling, so that's what I'd recommend using going forward.

We'll probably update the docs to use that syntax everywhere, because for now we show the two different syntaxes in different places of the Tailwind CSS documentation.
